third_party/packages/cupertino_icons/example/example.md

```diff
@@ -0,0 +1,3 @@
+Please find the usage of `cupertino_icons` here
+
+https://api.flutter.dev/flutter/cupertino/CupertinoIcons-class.html
```

Comment:
Markdown files allow text to be links, rather than embedding a full URL into the text. See the README for examples.

Reply:
I have added text links & a simple widget example in the markdown file.